hasher-matcher-actioner/src/OpenMediaMatch/app.py

```diff
@@ -96,12 +101,26 @@ def create_app() -> flask.Flask:
 
     scheduler: APScheduler | None = None
 
+    if _is_gunicorn() and (
+        app.config.get("ROLE_HASHER", False)
+        or app.config.get("ROLE_MATCHER", False)
+        or app.config.get("ROLE_CURATOR", False)
+    ) and (
+        app.config.get("TASK_INDEXER", False)
+        or app.config.get("TASK_FETCHER", False)
+    ):
+        # Only one Indexer and Fetcher task can be run in gunicorn workers -
+        # running those tasks in multiple workers can cause database conflicts
+        #
+        # Error out if gunicorn is our server and we think we've been misconfigured
+        raise RuntimeError("Cannot run Indexer or Fetcher tasks in gunicorn ROLE worker - must be run in separate deployments")
+
     with app.app_context():
+        scheduler = dev_apscheduler.get_apscheduler()
```

Comment:
What is the purpose of this change? If we aren't going to init the scheduler, why get it?